allPostsToWordpress.py

```diff
@@ -21,7 +21,12 @@ def addPost(post, channel, username, wpUrl, doProtected, protectedPassword, doCo
     ET.SubElement(item, 'wp:post_date_gmt').text = pubDate.astimezone(datetime.timezone.utc).strftime('%Y-%m-%d %H:%M:%S')
     ET.SubElement(item, 'wp:comment_status').text = 'open'
     ET.SubElement(item, 'wp:ping_status').text = 'open'
-    ET.SubElement(item, 'wp:post_name').text = post.get('linkId')
+    postTitle = post.find('title').text.strip().lower()
+    wordChar = re.sub('[^\s\w]', '', postTitle)
+    if postTitle == '(no subject)':
+        ET.SubElement(item, 'wp:post_name').text = post.get('linkId')
```

Comment:
Now I'm more confused! What I'm reading is that you're saying I should do something like this:

```python
    postName = if safeTitle == '(no subject)':
        postName.text = post.get('linkId')
    else:
        postName.text = postName

I'm sure that's not right but I can't understand what you're saying to mean anything else.